submit-queue/github/github.go

```diff
@@ -308,16 +305,28 @@ func ForEachCandidatePRDo(client *github.Client, user, project string, fn PRFunc
 
 		glog.V(8).Infof("%v", issue.Labels)
 		if !HasLabels(issue.Labels, []string{"lgtm", "cla: yes"}) {
+			glog.V(2).Infof("Skipping %d - doesn't have requisite labels", *pr.Number)
```

Comment:
seems like this may be very spammy for level 2?
also not sure that "requisite labels" is very meaningful without seeing the code.